### ISSUE TITLE
Book builder accessibility issues

### ISSUE THREAD

**Issue by yannisbarlas** [Coko-Foundation/ketty#640]
> **Migrated from GitLab**
> **Original Author:** gautier - editadapt (`@realgau`)
> **Original Link:** https://gitlab.coko.foundation/coko-org/products/ketty/ketty/-/issues/411

Using NVDA screen reader (https://www.nvaccess.org/) on book builder page :

* I can't access (using tab) to the elements "FRONTMATTER", "BODY", or "BACKMATTER", they are text elements alone, lost in a lot of links.
* The page has no Heading levels (h1, h2, etc.). 
* Elements like "upload word" "team manager", etc. are read but don't get focus. I wonder if those elements would deserve to be mentioned as NAV elements.

See the video for illustration (sorry for the echo and bad audio).

![Editoria_-_Mozilla_Firefox_2020-11-09_15-09-56-1](/uploads/bd28ac28465429a3912a4c3df305d8a5/Editoria_-_Mozilla_Firefox_2020-11-09_15-09-56-1.m4v)

Solutions may be : 
* use html semantics instead of divs (i believe frontmatter should be title of a section as in the final file) 
* add a better structure to the page (where can we cut content)
* use arialabels on buttons

(instance used for testing : editoria.cloud68.co)

*Note this is not a full accessibility review, only a quick look to see how it goes. *
